app/models/competition.rb

```diff
@@ -1933,10 +1914,10 @@ def persons_wcif(authorized: false)
       } },
       :wcif_extensions,
     ]
-    # V2 registrations store the event IDs in the microservice data, not in the monolith
-    includes_associations << :events unless self.uses_microservice_registrations?
 
-    registrations_relation = self.uses_microservice_registrations? ? self.microservice_registrations : self.registrations
+    includes_associations << :events
```

Comment:
The array `includes_associations` is literally being defined in the line directly above.
Now that the `if` / ternary check is not necessary anymore, you can just as well directly include `:events` in the list again